Incompatibility With Figura Mod on Forge

### Minecraft Version

1.20.1

### Oculus Version

oculus-mc1.20.1-1.6.15

### Rubidium / Embeddium Version

embeddium-0.3.1+mc1.20.1

### Operating System

win 11

### What is your GPU?

rtx 3070

### Java Version

jdk 17

### Reproduction Steps

launch game with both mods

### Crash Report file and latest.log

```shell
Exception caught from launcher
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:67)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:130)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:32)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:108)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
	... 8 more
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at MC-BOOTSTRAP/fmlloader@1.20.1-47.2.20/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111)
	at MC-BOOTSTRAP/fmlloader@1.20.1-47.2.20/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.clientService(CommonLaunchHandler.java:99)
	at MC-BOOTSTRAP/fmlloader@1.20.1-47.2.20/net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$makeService$0(CommonClientLaunchHandler.java:25)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
	... 15 more
Caused by: java.lang.ExceptionInInitializerError
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.resources.ResourceKey.m_135788_(ResourceKey.java:26)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.core.registries.Registries.m_257397_(Registries.java:200)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.core.registries.Registries.<clinit>(Registries.java:93)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.world.flag.FeatureElement.<clinit>(FeatureElement.java:18)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.world.level.block.Blocks.<clinit>(Blocks.java:45)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.world.item.Items.<clinit>(Items.java:21)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.client.renderer.entity.ItemRenderer.<clinit>(ItemRenderer.java:51)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.client.renderer.RenderType.<clinit>(RenderType.java:101)
	at TRANSFORMER/figura@0.1.2+1.20.1/forge.org.figuramc.figura.model.rendering.texture.RenderTypes.<clinit>(RenderTypes.java:27)
	at TRANSFORMER/figura@0.1.2+1.20.1/forge.org.figuramc.figura.lua.docs.FiguraListDocs$2.<init>(FiguraListDocs.java:44)
	at TRANSFORMER/figura@0.1.2+1.20.1/forge.org.figuramc.figura.lua.docs.FiguraListDocs.<clinit>(FiguraListDocs.java:43)
	at TRANSFORMER/minecraft@1.20.1/com.mojang.blaze3d.platform.InputConstants$Type.handler$gok000$addKey(InputConstants.java:515)
	at TRANSFORMER/minecraft@1.20.1/com.mojang.blaze3d.platform.InputConstants$Type.m_84899_(InputConstants.java)
	at TRANSFORMER/minecraft@1.20.1/com.mojang.blaze3d.platform.InputConstants$Type.<clinit>(InputConstants.java:214)
	at TRANSFORMER/oculus@1.6.15/net.coderbot.iris.Iris.<clinit>(Iris.java:95)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.SystemReport.handler$bhm000$fillSystemDetails(SystemReport.java:523)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.SystemReport.<init>(SystemReport.java:58)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.CrashReport.<init>(CrashReport.java:31)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.CrashReport.m_127529_(CrashReport.java:216)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.client.main.Main.main(Main.java:149)
	... 23 more
Caused by: java.lang.IllegalArgumentException: Not bootstrapped (called from registry ResourceKey[minecraft:root / minecraft:root])
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.Bootstrap.m_179916_(Bootstrap.java:113)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.Bootstrap.m_179912_(Bootstrap.java:106)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.core.MappedRegistry.<init>(MappedRegistry.java:85)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.core.MappedRegistry.<init>(MappedRegistry.java:81)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.core.registries.BuiltInRegistries.<clinit>(BuiltInRegistries.java:113)
	... 43 more
```


### Additional context

issue reported on Figura github [here](https://github.com/FiguraMC/Figura/issues/176)

--- a/src/main/java/net/coderbot/iris/Iris.java
+++ b/src/main/java/net/coderbot/iris/Iris.java
@@ -92,10 +92,11 @@ public class Iris {
 	private static PipelineManager pipelineManager;
 	private static IrisConfig irisConfig;
 	private static FileSystem zipFileSystem;
-	private static final KeyMapping reloadKeybind = new KeyMapping("iris.keybind.reload", InputConstants.Type.KEYSYM, GLFW.GLFW_KEY_R, "iris.keybinds");
-	private static final KeyMapping toggleShadersKeybind = new KeyMapping("iris.keybind.toggleShaders", InputConstants.Type.KEYSYM, GLFW.GLFW_KEY_K, "iris.keybinds");
-	private static final KeyMapping shaderpackScreenKeybind = new KeyMapping("iris.keybind.shaderPackSelection", InputConstants.Type.KEYSYM, GLFW.GLFW_KEY_O, "iris.keybinds");
-	private static final KeyMapping wireframeKeybind = new KeyMapping("iris.keybind.wireframe", InputConstants.Type.KEYSYM, InputConstants.UNKNOWN.getValue(), "iris.keybinds");
+
+	private static KeyMapping reloadKeybind;
+	private static KeyMapping toggleShadersKeybind;
+	private static KeyMapping shaderpackScreenKeybind;
+	private static KeyMapping wireframeKeybind;
 
 	private static final Map<String, String> shaderPackOptionQueue = new HashMap<>();
 	// Flag variable used when reloading
@@ -132,6 +133,11 @@ public class Iris {
 	 * <p>This is called right before options are loaded, so we can add key bindings here.</p>
 	 */
 	public static void onEarlyInitialize() {
+		reloadKeybind = new KeyMapping("iris.keybind.reload", InputConstants.Type.KEYSYM, GLFW.GLFW_KEY_R, "iris.keybinds");
+		toggleShadersKeybind = new KeyMapping("iris.keybind.toggleShaders", InputConstants.Type.KEYSYM, GLFW.GLFW_KEY_K, "iris.keybinds");
+		shaderpackScreenKeybind = new KeyMapping("iris.keybind.shaderPackSelection", InputConstants.Type.KEYSYM, GLFW.GLFW_KEY_O, "iris.keybinds");
+		wireframeKeybind = new KeyMapping("iris.keybind.wireframe", InputConstants.Type.KEYSYM, InputConstants.UNKNOWN.getValue(), "iris.keybinds");
+
 		try {
 			if (!Files.exists(getShaderpacksDirectory())) {
 				Files.createDirectories(getShaderpacksDirectory());

